t/03_smtp_nb.t

```diff
@@ -525,6 +525,57 @@ $loop->reactor->remove($sock);
 close $sock;
 kill 15, $pid;
 
+# 9 - AUTH LOGIN method
+my ($pid, $sock, $host, $port) = Utils::make_smtp_server();
```

Comment:
my is useless here and below, this variables were declared above

Reply:
Sorry again, I copy-n-pasted your first test to get mine working, but forget to remove `my`.